src/IssueViz/Helpers/ShowInBrowserService.cs

```diff
@@ -33,8 +32,6 @@ public interface IShowInBrowserService
 
         void ShowDocumentation();
 
-        void ShowRuleDescription(string ruleKey);
```

Comment:
No longer required. The navigation command now calls `IEducation` instead, which (once the changes in #3652 have been merged), will handle showing help in the IDE or the browser.